org-static-blog.el

```diff
@@ -182,6 +182,16 @@ See also `org-static-blog-preview-ellipsis' and
   :type '(boolean)
   :safe t)
 
+(defcustom org-static-blog-preview-start nil
+  "Marker indicating the beginning of a post's preview."
+  :type '(choice (const :tag "First paragraph" nil) (string))
+  :safe t)
+
+(defcustom org-static-blog-preview-end nil
+  "Marker indicating the end of a post's preview."
```

Comment:
This should probably include a note about the meaning of `nil`, much like in `org-static-blog-rss-max-entries`. I'm assuming if `nil`, the "legacy" behavior of the first paragraph is used?

What happens if only one of `org-static-blog-preview-start` and `org-static-blog-preview-end` is set?